sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -24,6 +24,14 @@ The Azure Monitor Distro is a distribution of the .NET OpenTelemetry SDK with in
 
 * [Logs](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/docs/logs/getting-started-console)
 
+* Resource Detectors
+  * **AppServiceResourceDetector**: Adds resource attributes for the applications running in Azure App Service.
+  * **AzureVMResourceDetector**: Adds resource attributes for the applications running in an Azure Virtual Machine.
+  * **AzureContainerAppsResourceDetector**: Adds resource attributes for the applications running in Azure Container Apps.
```

Comment:
nit: elsewhere in this doc we link to external packages. Should we link to the contrib repo for these classes?
I'm thinking not right now because they are vendored.
In the future, would we want to link to the contrib repo?

Reply:
I prefer to add the contrib repo link once the vendored code is removed.

Reply:
As we mention in SQL instrumentation, it might be appropriate to specify that it is vendored code. What are your thoughts?

Reply:
I think mentioning that it is vendored is appropriate. That will also serve as our reminder to update when stable. :)